### PR TITLE
Gravity TOP pulls content up

### PR DESCRIPTION
**Pull request by elek90**
If using a theme like Theme.AppCompat.Light.NoActionBar, the getStatusBarHeightInPx function returns a negative value which then pulls the content of the flashbar up.

This could also be what's causing #3

### CODE REVIEW

**Comment by ptrkaz** [comment]
Merge it plox

**Comment by elek90** [comment]
Sorry, can't merge it since I don't have write access.

@aritraroy, can you have a look at this? :)

**Review by aritraroy — APPROVED** [review]
Thanks for the fix.

**Comment by aritraroy** [comment]
Done.

**Comment by ptrkaz** [comment]
Thank you, that will be very useful to my current project.

**Comment by ptrkaz** [comment]
When will it be usable from gradle import?

**Comment by aritraroy** [comment]
You should be able to use the latest `1.0.3` as a Gradle import. Please check and let me know.

**Comment by klevisndoka** [comment]
1.0.3 works as it should for me.

**Comment by paradox02** [comment]
for me not work, when want show snack bar in the top.. snackbar show, but status bar its on it. 
version 1.0.3

**Comment by findyourexit** [comment]
I moved to version 1.0.3 to see if it fixed the Gravity.TOP issue, but it seems to behave the same as 1.0.2.

**Comment by gaborbiro** [comment]
I worked around it by wrapping the showing of Flashbar in `window.decorView.post {...}`.
Problem is that when the Flashbar is shown too early in the Activity lifecycle, `statusBarHeight` is still 0.

**Comment by rkoshti** [comment]
just append \n\n in the message it works for me surprisingly...!

**Pull request merged**
